buildSrc/src/main/kotlin/io/spine/examples/shareaware/dependency/JavaX.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.shareaware.dependency
+
+object JavaX {
```

Comment:
I am not sure a) how is this declaration related to this PR, and b) why we need it at all.

Please review.

Reply:
I am not sure but I think we need it because `Spine` using `@javax.annotation.Generated("by Spine Model Compiler")` this annotation in generated code comes from the `JavaX` library.
![image](https://user-images.githubusercontent.com/44636928/223676173-861b6445-fd31-4f90-b29f-2e4ae77ef3ac.png)
I have discovered that we may need the `JavaX` library in this PR because we generate some code here.
Anyway, I am not sure about this, please correct me if I am wrong.

Reply:
I have this type as a part of JDK 8. It's in my `rt.jar`:
![Screenshot 2023-03-08 at 12 52 28](https://user-images.githubusercontent.com/82468/223718066-02649be0-3339-4b13-a773-04f3cca399db.png)

Reply:
Yes, the problem was in JDK, sorry for that.